### DISCUSSION TITLE
Release notes to version 0.6

### DISCUSSION THREAD

**Discussion by faerbersteve** [unikraft/unikraft#363] [General] [answered]
Hey,
thanks for your work on this project.
I saw that you have released a new version.
I am curious about the changes.

May I ask where I can find the release notes to this new version? 

Thanks

**Comment by nderjung** [comment, marked as answer]
Hey @faerbersteve! We'll be posting an update to the website shortly, but for now the release note can be found [here](https://github.com/unikraft/unikraft/releases/tag/RELEASE-0.6). :)

**Reply by faerbersteve** [reply]
thanks for the quick response

**Comment by razvand** [comment]
@faerbersteve, since we are already (viciously) using [the Unikraft Discord server](https://bit.ly/UnikraftDiscord) for discussions, meetings, brainstorming and jokes (mostly bad, we do what we can), we will close the Discussions tab. We will do this on Friday, March 31, 2023.

Please move any discussions or questions on [the Unikraft Discord server](https://bit.ly/UnikraftDiscord). See you all there! :handshake:
